_2021-finalproject/README.md

```diff
@@ -1,63 +1,63 @@
-# Final Project

```

Comment:
Your commits sholdn't touch this file

Reply:
Is it I should cancel my commit ?

Reply:
but if I cann't touch the file ,where I can make the change to solve the conflict?in the resolved?

Reply:
No, here shows the finally changes that your commits made, not one single commit.

Reply:
emmm,  Shouldn't I modify it in Git? I am confused now .so  how I can through the check?Cancle my commits of about modify the _2021-finalproject/README.md? But how can I resolve my conflict without modifying the file?
@MuZhou233

Reply:
You can get inspire from other PR, you are not the only one face to this problem.